src/conversions.h

```diff
@@ -0,0 +1,39 @@
+#include <vector>
+#include <iostream>
+#include <cstdint>
+#include <opencv2/opencv.hpp>
+#include <opencv2/core/core.hpp>
+#include "feature_lib.h"
+
+#ifndef FEATUREDETECT_SRC_CONVERSIONS_H_
+#define FEATUREDETECT_SRC_CONVERSIONS_H_
+cv::Mat VectorToMat(const std::vector<std::uint8_t> &inputVector, int Height, int Width) {
```

Comment:
>Actually, all function names in this file don't, so fix them all, please
public:
  int CountFooErrors(const std::vector<Foo>& foos) {
that example from google C++ codestyle link
VectorToMat matches, isn't it?

Reply:
Yep, you are right here

Reply:
Actually, let's make an exception here and name functions in `camelCase` with leading lowercase letter, because we intend to generate an interface for Java where all class members will have the same names as they do in C++.

Reply:
okay, should i rename whole project functions? or only in conversions file?

Reply:
In fact, only the public ones will be ported, but let's rename all of them for the uniformity